guide/06-imagery-and-raster-analysis/orthomapping_guide_1.ipynb

```diff
@@ -0,0 +1,741 @@
+{
```

Comment:
<p>Can you rename variable names so they follow PEP8 coding standards? this applies for the entire notebook series.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>